<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hospedes</class>
 <widget class="QDialog" name="Hospedes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btNovoHospede">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar Novo Hospede</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primeiro Nome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Idade:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>RPG:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textNome">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textIdade">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textRPG">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="R3Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>3 dias</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="R7Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>96</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>7 dias </string>
   </property>
  </widget>
  <widget class="QRadioButton" name="R12Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>96</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>12 dias </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantos dias o cliente ira passar?</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="R15Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>96</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>15 dias</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbteste">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ultimo nome:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textSobrenome">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
